src/components/charts/MultiChartSummary.tsx

```diff
@@ -1,200 +1,328 @@
+    configureChartIndex: number
+    hideDeployedValues?: boolean
+    name?: string
+    setChartDetailsUpdate?: React.Dispatch<React.SetStateAction<boolean>>
 }
 
-const MultiChartSummary: React.FC<MultiChartSummaryProps> = ({ charts, getChartVersionsAndValues, toggleChart, configureChart, handleChartValueChange, handleChartVersionChange, chartListing, configureChartIndex, removeChart, hideDeployedValues }) => {
-
+const MultiChartSummary: React.FC<MultiChartSummaryProps> = ({
+    charts,
+    getChartVersionsAndValues,
+    toggleChart,
+    configureChart,
+    handleChartValueChange,
+    handleChartVersionChange,
+    chartListing,
+    configureChartIndex,
+    removeChart,
+    hideDeployedValues,
+    name,
+    setChartDetailsUpdate,
+}) => {
     return (
         <div className="chart-group--summary">
-            {chartListing && <div className={`selected-chart-widget p-12 select-chart cursor ${typeof configureChartIndex !== 'number' ? 'active' : ''}`} style={{ gridTemplateColumns: "40px 1fr 24px" }} onClick={chartListing}>
-                <Forward className="rotate anchor" style={{ ['--rotateBy' as any]: '180deg' }} />
-                <div className="anchor ml-18">Select Charts</div>
-                <span />
-            </div>}
-            {chartListing && <div className="chart-count flex left column">
-                <b>{charts.length} charts selected</b>
-                <span>Set default chart version and values for each chart.</span>
-            </div>}
-            {!chartListing && charts.length === 0 && <div className="flex column" style={{ height: '100%' }}>
-                <EmptyFolder />
-                <p>No charts here.</p>
-                <p>Edit group to add charts.</p>
-            </div>}
-            {charts.length > 0 && <div className="overflow selected-widgets-container">
-                {charts?.map((chart: ChartGroupEntry, index) =>
-                    <SelectedChartWidget
-                        selectChart={configureChart ? e => configureChart(index) : null}
-                        selectChartVersion={handleChartVersionChange ? e => { handleChartVersionChange(index, e.target.value) } : null}
-                        selectChartValues={({ kind, id }) => id === chart.appStoreValuesVersionId ? noop : handleChartValueChange(index, kind, id)}
-                        key={index}
-                        remove={removeChart ? e => removeChart(index) : null}
-                        toggleChart={toggleChart && configureChartIndex !== index ? e => toggleChart(index) : null}
-                        chart={chart}
-                        selected={configureChartIndex === index}
-                        getChartVersionsAndValues={getChartVersionsAndValues ? () => getChartVersionsAndValues(chart.id, index) : null}
-                        hideDeployedValues={hideDeployedValues}
-                    />
-                )
-                }
-            </div>}
+            {chartListing && (
```

Comment:
Can we wrap all these in single condition?

Reply:
Done